Gruntfile.js

```diff
@@ -194,6 +194,7 @@ module.exports = function (grunt) {
           styles: ['node_modules/datatables.net-dt/css/jquery.dataTables.css',
             'node_modules/patternfly/dist/css/patternfly.css',
             'node_modules/patternfly/dist/css/patternfly-additions.css',
+            'http://rawgit.com/junezhang/showcase/master/dist/styles/patternfly-showcase.css',
```

Comment:
+1 

Each repo should have it's own get started and contribute pages, specific with details and information for that particular repo. June has already populated the one for Angular 1 here: http://rawgit.com/junezhang/showcase/master/index.html

@jeff-phillips-18 @dtaylor113 does this information look up to date and correct? If so, @dabeng can we introduce this to the angular repo with this PR?